packages/core/http/core-http-common/src/constants.ts

```diff
@@ -8,5 +8,6 @@
 
 /** @public */
 export const ELASTIC_HTTP_VERSION_HEADER = 'elastic-api-version' as const;
+export const ELASTIC_HTTP_VERSION_QUERY_PARAM = 'apiVersion' as const;
```

Comment:
For consistency with the version header, 
```suggestion
export const ELASTIC_HTTP_VERSION_QUERY_PARAM = 'elasticApiVersion' as const;
```